src/FSAL/FSAL_CORTXFS/conf/nfs_setup.sh

```diff
@@ -409,6 +411,14 @@ Options:
   -p Prompt
   -q To perform Setup on Provisioner VM
   -d To create FS
+     Note: While using -d, the FS name is mandatory. The FS Block size is an optional
```

Comment:
provide info on allowed bsize .. like min, max & multiple of xyz

Reply:
Addressed.